package.json

```diff
@@ -18,28 +18,36 @@
     "eyeglass-module"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
   },
   "dependencies": {
-    "chalk": "^2.4.2",
+    "chalk": "^4.0.0",
     "css": "^2.2.1",
     "lodash.find": "^4.6.0",
     "lodash.foreach": "^4.5.0",
     "lodash.last": "^3.0.0"
   },
   "devDependencies": {
-    "beeper": "^2.0.0",
     "chai": "^4.1.2",
-    "coveralls": "^3.0.3",
-    "fancy-log": "^1.3.3",
-    "gulp": "^4.0.2",
-    "gulp-sass-lint": "^1.3.4",
-    "jest": "^24.8.0",
-    "mocha": "^6.1.4",
-    "node-sass": "^4.12.0",
-    "nyc": "^14.1.1",
-    "sassdoc": "^2.7.0",
-    "sassdoc-theme-herman": "^3.0.2"
+    "coveralls": "^3.0.11",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-prettier": "^3.1.2",
+    "jest": "^25.3.0",
+    "mocha": "^7.1.1",
+    "node-sass": "^4.13.1",
```

Comment:
@mirisuzanne Do we want to run the tests on both dart-sass and node-sass? Might be a good idea, if we claim to support both... Otherwise we can remove this node-sass dependency, I think.

Reply:
It's good conceptually, but I also updated the internal syntax. So once we merge my changes, Dart-Sass will be required for the upgrade (until node-sass supports modules).

We could split out your changes from mine, if we want to release a minor update that supports Dart Sass without dropping node-sass support?

Reply:
@mirisuzanne Hm, okay -- I'm not sure a new release is required then. True _already_ supported both node-sass and dart-sass, right? But in that case I should make a few changes here to make it clear we're dropping node-sass support entirely.

Reply:
ah, good point. Yeah, go for it.

Reply:
Ok, removed support for node-sass.